src/core/timer.h

```diff
@@ -41,6 +41,7 @@ struct Timer {
 
         usec_t accuracy_usec;
         usec_t random_usec;
+        usec_t fixed_delay_usec;
```

Comment:
I'd not store this as a field. It's cheap to recalc it whenever we need it, and dropping the field simplifies things, since we don't need to keep anything correctly updated.

i.e. change timer_set_fixed_delay() to return the pseudo-random value, and call it from add_random()